### DISCUSSION TITLE
Point of Origin

### DISCUSSION THREAD

**Discussion by Misfualt** [Povstalec/StargateJourney#103] [Q&A] [answered]
So I'm playing with the mod with in atm9 and I'm trying to code in a different point of origin with the cc:Tweaked mod. but every time i type in a different point of origin name like "destiny" or "terra" it changes the point of origin to an error and i cant get it back. does everyone know how to change the point of origin for the Pegasus gates? and if at all possible the symbols too?
overridePointOfOrigin(String pointOfOrigin) this is the code I'm using trying to override the point of origin symbol.

**Comment by lukaskabc** [comment, marked as answer]
Use full name `sgjourney:abydos`, available PoOs can be found [here](https://github.com/Povstalec/StargateJourney/tree/main/src/main/resources/data/sgjourney/sgjourney/point_of_origin) + those added by datapacks (if used)

Same for symbols `sgjourney:pegasus`, `sgjourney:milky_way`, `sgjourney:universal` ([list](https://github.com/Povstalec/StargateJourney/tree/main/src/main/resources/data/sgjourney/sgjourney/symbols)) + those added by datapacks (if used)
The symbols displayed may vary depending on the client configuration (depends if symbols or symbol_set is used).

**Comment by Povstalec** [comment]
Just so I'm not repeating what lukasbc already said while still adding to the conversation:

Since anyone can add custom Points of Origin with the use of datapacks, every Point of Origin has a MODID prefix, same as blocks, entities and everything else in-game.

So instead of writing "destiny" or "terra", which, due to the way Minecraft works, will default to "minecraft:destiny" and "minecraft:terra" you need to specify "sgjourney:destiny" and "sgjourney terra". If you wanted to use Points of Origin from other mods/datapacks, you would also write them with the prefix of the mod/datapack.

Also a pretty important thing to note is that you must not forget to first set [dynamicSymbols(false)](https://github.com/Povstalec/StargateJourney/wiki/ComputerCraft#connected-to-a-pegasus-stargate) otherwise the custom Symbols and Point of Origin you set will be reset next time the gate loads.
